api/openapi-spec/swagger.json

```diff
@@ -3701,6 +3701,10 @@
      "bridge": {
       "$ref": "#/definitions/v1.InterfaceBridge"
      },
+     "model": {
+      "description": "Interface model supported by libvirt\nDefaults to virtio",
```

Comment:
referencing libvirt here might be a little confusing since we don't mention libvirt anywhere else in the api

Reply:
Could it actually be that https://github.com/kubevirt/kubevirt/issues/1085 could help us here?
Aka have an enum to outline all the possible options.

We should fix #1085 in order to let the API include all the options that are available (aka enable user introspection) otherwise we'll have to list all the details in the user guide, which is not intended.

Reply:
@fabiand I don't think we can easily add support for enums on our own, there are kubernetes / opengen-api issues that I referred in our issue that should tackle it though. I could spend time on these issues in relevant projects but is it something we want to focus on right now? I would prefer if we just track progress in opengen-api repo and maybe help with reviews, testing, etc. but move forward on this PR. Thoughts?